daml-lf/spec/daml-lf-1.rst

```diff
@@ -1609,6 +1698,15 @@ Specifically, a template definition is *coherent* if:
 * Its argument data type is not an argument to any other template.
 
 
+Exception coherence
+~~~~~~~~~~~~~~~~~~~
+
+The *exception coherence* condition is literally the same as the template
+coherence condition with "template" replaced by "exception". Note that there is
+no condition precluding a type from having an associated template definition as
+well as an associated exception definition.
```

Comment:
That seems a bit fishy. Should it be restricted? We shouldn't be generating "exception templates" anyway in the compiler, but it's a vector for unwelcome weird edge cases.

Reply:
Yeah, I was on the fence regarding that one. Let's be restrictive at the beginning. I'll reword accordingly.